src/SDK/Internal/Conversion/AccessibleAccountResponseConverter.cs

```diff
@@ -0,0 +1,58 @@
+using System;
```

Comment:
I am not sure I understand the use of that class. You create it with a mutable object so it could store calls responses ? Why don't you just return a new object each time you do a call ?

Reply:
This class is just named as "Response", it's not a REST Response.

Reply:
Ok it's one those converter, I thought it was something else. This is fine then.